### PR TITLE
chore(redux): convert preferences to redux slice

### PR DESCRIPTION
**Pull request by mperdomo-bc**
I'm getting the slight idea that many of these are not used - as they clash with some things that are in [blockchain-wallet-v4-frontend/src/data/modules/settings/reducers.js](https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/e0088acb5fab8eaff293429e8589f44fae9f001e/packages/blockchain-wallet-v4-frontend/src/data/modules/settings/reducers.js)

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/Chart/selectors.ts [hunk 1]
```diff
@@ -7,23 +7,21 @@ import { selectors } from 'data'
 export const getData = createDeepEqualSelector(
   [
     selectors.core.settings.getCurrency,
-    selectors.preferences.getPriceChart,
+    selectors.components.priceChart.getTime,
```
Comment (mperdomo-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/Chart/selectors.ts:10]: `preferences.getPriceChart` was updated, via the `preferences` reducer, with the same actions used in the `priceChart` reducer - so in this case the data obtained was exactly the same

**Comment by sstephanou-bc** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [18f8a1c4\-552b\-4fc7\-8031\-94dd7e5b1bb5](https://eu.ast.checkmarx.net/projects/f3c879d8-da6e-4aaa-aeee-f377cc306d79/scans?id=18f8a1c4-552b-4fc7-8031-94dd7e5b1bb5&branch=redux-slice-preferences)
### No New Or Fixed Issues Found

File: packages/blockchain-wallet-v4-frontend/src/data/actions.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ import * as ws from './middleware/webSocket/coins/actions'
 import { actions as misc } from './misc/slice'
 import { actions as modals } from './modals/slice'
 import * as modules from './modules/actions'
-import * as preferences from './preferences/actions'
+import { actions as preferences } from './preferences/preferencesSlice'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/actions.ts]: naming convention is to name file just `slice` please check import below this line

File: packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Preferences/CryptoLinkHandling/index.js [hunk 3]
```diff
@@ -78,7 +90,7 @@ class CryptoLinkHandlingContainer extends React.PureComponent {
                         <a
                           href='https://caniuse.com/?search=registerProtocolHandler'
                           target='_blank'
-                          rel='noopener noreferrrer noreferrer'
+                          rel='noopener noreferrer'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Preferences/CryptoLinkHandling/index.js:93]: nice 👍

File: packages/blockchain-wallet-v4-frontend/src/data/actions.ts (hunk 2 above)
Comment (mperdomo-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/actions.ts]: I did this for ease of search - when searching a file name it's easier to do with a meaningful name. I know I could also search for `preferences/slice` but that's only if the editor/IDE used allows to search file paths or folder names besides files.

Comment (mperdomo-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/actions.ts]: Will revert this for the sake of getting it approved and consistency

**Review by jjBlockchain — APPROVED** [review]

**Pull request merged**